src/static/js/pad_cookie.js

```diff
@@ -79,6 +80,10 @@ var padcookie = (function()
       alreadyWarnedAboutNoCookies = true;
     }
   }
+  
+  function isHttpsScheme() {
+    return window.location.protocol == "https:";
```

Comment:
I think this won't work for reverse proxys as they usually forward requests over HTTP. You should honor the [`X-Forwarded-Proto`](https://stackoverflow.com/questions/13111080/what-is-a-full-specification-of-x-forwarded-proto-) header - and probably also the standardized [`Forwarded`](https://tools.ietf.org/html/rfc7239) header, but AFAIK the unofficial `X-Forwarded-Proto` is still more common.